content/docs/latest/installing/vms/hyper-v.md

```diff
@@ -0,0 +1,119 @@
+---
+title: Running Flatcar Container Linux on Hyper-V
+title: Running on Hyper-V
+weight: 30
+aliases:
+    - ../../os/booting-on-hyper-v
+    - ../../cloud-providers/booting-on-hyper-v
```

Comment:
I think we don't need these because they were added for keeping existing links working but this here is a new document.